### PR TITLE
AssetTypes Detail page

### PR DESCRIPTION
**Pull request by Manu92915**
AssetTypes Detail page created

### CODE REVIEW

File: src/Web/Client/Pages/Settings/AssetTypesDetail.razor.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿.icon-container{
+    display: flex;
```
Comment (SidiqovAbbos) [inline src/Web/Client/Pages/Settings/AssetTypesDetail.razor.css]: We already have a class 'd-flex'

https://mudblazor.com/utilities/enable-flex#enabling-flexbox

File: src/Web/Client/Shared/AssetTypesLayout.razor [hunk 2]
```diff
@@ -0,0 +1,58 @@
+﻿@inherits LayoutComponentBase
```
Comment (SidiqovAbbos) [inline src/Web/Client/Shared/AssetTypesLayout.razor]: Why did you create a new layout class when we have Settings Layout?

File: src/Web/Client/Services/AssetTypes/AssetTypesService.cs [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+        public event Action OnChange;
+
+        public string GenerateShortName(string name)
+        {
+            return ContractString(StringWithoutVowels(name));
+        }
+
+        public async Task<AssetType> GetAssetTypeById(string id)
+        {
+            var response = await _httpClient.GetFromJsonAsync<List<AssetType>>("https://localhost:7098/assettypes");
```
Comment (SidiqovAbbos) [inline src/Web/Client/Services/AssetTypes/AssetTypesService.cs]: This port doesn't work for me!

File: src/Web/Client/_Imports.razor [hunk 4]
```diff
@@ -9,3 +9,7 @@
 @using Web.Client
 @using Web.Client.Shared
 @using MudBlazor
```
Comment (SidiqovAbbos) [inline src/Web/Client/_Imports.razor:11]: There are some warnings. Please fix also these warnings

File: src/Web/Client/Shared/AssetTypesLayout.razor (hunk 2 above)
Comment (Manu92915) [reply src/Web/Client/Shared/AssetTypesLayout.razor]: Because I thought it was needed for something else

**Review by SidiqovAbbos — APPROVED** [review]

**Pull request merged**